scripts/launch/demo.sh

```diff
@@ -141,14 +148,40 @@ while [[ $i -lt $panesCnt ]]; do
     if [[ $WALLET_TEST != "" ]] && [[ $i == $((n-1)) ]]; then
         updater_file="$config_dir/updater$i.sh"
         launcher_=$(find_binary cardano-launcher)
-        launcher_args="--node $node_ "
+
+        # TODO: make sure that this does exactly what is intended
```

Comment:
@georgeee can you take a look at this and the next TODO and make sure that exactly what we want happens here?